mis_builder/models/aep.py

```diff
@@ -281,14 +284,20 @@ def do_queries(self, date_from, date_to,
                                         ['debit', 'credit', 'account_id'],
                                         ['account_id'])
             for acc in accs:
+                company = account_model.browse(acc['account_id'][0]).company_id
```

Comment:
This browse inside the loop is bad for performance.

I suggest:
1. fetching and computing currency rates by company at the beginning of the method, outside of the loops, store that in a dictionary
2. adding company_id in the read_group